<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
********************************************************************************
**
** Copyright (c) 2017 The University of Notre Dame
** Copyright (c) 2017 The Regents of the University of California
**
** Redistribution and use in source and binary forms, with or without modification,
** are permitted provided that the following conditions are met:
**
** 1. Redistributions of source code must retain the above copyright notice, this 
** list of conditions and the following disclaimer.
**
** 2. Redistributions in binary form must reproduce the above copyright notice, this
** list of conditions and the following disclaimer in the documentation and/or other
** materials provided with the distribution.
**
** 3. Neither the name of the copyright holder nor the names of its contributors may
** be used to endorse or promote products derived from this software without specific
** prior written permission.
**
** THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS &quot;AS IS&quot; AND ANY
** EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
** OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
** SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
** INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
** TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
** BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
** CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
** IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
** SUCH DAMAGE.
**
**********************************************************************************

// Contributors:
 </comment>
 <class>CWE_MainWindow</class>
 <widget class="QMainWindow" name="CWE_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFD Client Program</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="HeaderWidget" name="header">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedPageContainer">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabContainer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="CWE_welcome_screen" name="tab_welcome_screen">
           <attribute name="title">
            <string>Welcome</string>
           </attribute>
          </widget>
          <widget class="CWE_help" name="tab_help">
           <attribute name="title">
            <string>Help</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_spacer_1">
           <attribute name="title">
            <string>Page text space</string>
           </attribute>
          </widget>
          <widget class="CWE_landing" name="tab_landing_page">
           <attribute name="title">
            <string>Dashboard</string>
           </attribute>
          </widget>
          <widget class="CWE_Create_Copy_Simulation" name="tab_create_new">
           <attribute name="title">
            <string>Create Job</string>
           </attribute>
          </widget>
          <widget class="CWE_manage_simulation" name="tab_manage_and_run">
           <attribute name="title">
            <string>Case Select</string>
           </attribute>
          </widget>
          <widget class="CWE_file_manager" name="tab_files">
           <attribute name="title">
            <string>Files</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_spacer_2">
           <attribute name="title">
            <string>some text for size</string>
           </attribute>
          </widget>
          <widget class="CWE_Parameters" name="tab_parameters">
           <attribute name="title">
            <string>Parameters</string>
           </attribute>
          </widget>
          <widget class="CWE_Results" name="tab_results">
           <attribute name="title">
            <string>Results</string>
           </attribute>
          </widget>
          <widget class="CWE_Debug_Widget" name="tab_debug">
           <attribute name="title">
            <string>.</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="SideBar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="stackContainer" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="FooterWidget" name="footer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_existing"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="action_Landing_Page"/>
    <addaction name="actionCreate_New_Simulation"/>
    <addaction name="actionManage_Simulation"/>
    <addaction name="actionManage_and_Download_Files"/>
    <addaction name="actionManage_Remote_Jobs"/>
    <addaction name="actionTutorials_and_Help"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CWE"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionCreate_New_Simulation">
   <property name="text">
    <string>Create &amp;New Simulation</string>
   </property>
  </action>
  <action name="actionManage_Simulation">
   <property name="text">
    <string>Manage and &amp;Run Simulation</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen_existing">
   <property name="text">
    <string>&amp;Open existing</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionAbout_CWE">
   <property name="text">
    <string>&amp;About SimCenter CWE</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Landing_Page">
   <property name="text">
    <string>&amp;Landing Page</string>
   </property>
  </action>
  <action name="actionManage_Jobs">
   <property name="text">
    <string>Manage and &amp;Download Files</string>
   </property>
  </action>
  <action name="actionManage_Remote_Jobs">
   <property name="text">
    <string>Manage Remote &amp;Jobs</string>
   </property>
  </action>
  <action name="actionTutorials_and_Help">
   <property name="text">
    <string>Tutorials and &amp;Help</string>
   </property>
  </action>
  <action name="actionManage_and_Download_Files">
   <property name="text">
    <string>Manage and &amp;Download Files</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderWidget</class>
   <extends>QFrame</extends>
   <header>../AgaveExplorer/SimCenterCommon/HeaderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FooterWidget</class>
   <extends>QFrame</extends>
   <header>../AgaveExplorer/SimCenterCommon/FooterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_landing</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_landing.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_help</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_help.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_file_manager</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_file_manager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_manage_simulation</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_manage_simulation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_welcome_screen</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_welcome_screen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_Results</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_results.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_Parameters</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_parameters.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_Create_Copy_Simulation</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_create_copy_simulation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_Debug_Widget</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_debug_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
